tests/providers/yandex/secrets/test_secrets_manager.py

```diff
@@ -0,0 +1,343 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+import json
+from unittest.mock import Mock, patch
+
+import pytest
+import yandex.cloud.lockbox.v1.payload_pb2 as payload_pb
+import yandex.cloud.lockbox.v1.secret_pb2 as secret_pb
+import yandex.cloud.lockbox.v1.secret_service_pb2 as secret_service_pb
+
+from airflow.providers.yandex.hooks.yandex import YandexCloudBaseHook
+
+try:
+    import yandex
+    import yandexcloud
+
+    from airflow.providers.yandex.secrets.secrets_manager import SecretsManagerBackend
+except ImportError:
+    yandex = None
+    yandexcloud = None
```

Comment:
I think this block is redundant, maybe it exists from the past when yandex provider do not preinstall or have some issue with ARM image in the past (difficult to guess).

You could get rid of try-except block as well as  `pytest.mark.skipif` bellow

```suggestion
import yandex
import yandexcloud
from airflow.providers.yandex.secrets.secrets_manager import SecretsManagerBackend
```